CMakeLists.txt

```diff
@@ -949,15 +950,18 @@ if(EXPERIMENTAL)
   set(PYBIND11_FINDPYTHON OFF)
   set(MANIFOLD_PYBIND OFF)
   set(MANIFOLD_TEST OFF)
-
-  if(CMAKE_UNITY_BUILD)
-    set(CMAKE_UNITY_BUILD OFF)
-    add_subdirectory(submodules/manifold EXCLUDE_FROM_ALL)
-    set(CMAKE_UNITY_BUILD ON)
+  if(USE_BUILTIN_MANIFOLD)
+    if(CMAKE_UNITY_BUILD)
+      set(CMAKE_UNITY_BUILD OFF)
+      add_subdirectory(submodules/manifold EXCLUDE_FROM_ALL)
+      set(CMAKE_UNITY_BUILD ON)
+    else()
+      add_subdirectory(submodules/manifold EXCLUDE_FROM_ALL)
+    endif()
   else()
-    add_subdirectory(submodules/manifold EXCLUDE_FROM_ALL)
+    find_package(manifold REQUIRED)
   endif()
-  target_link_libraries(OpenSCAD PRIVATE manifold polygon)
+  target_link_libraries(OpenSCAD PRIVATE manifold polygon cross_section)
```

Comment:
Are anyone already packaging Manifold and provide CMake or pkg-config configs for this? I'd expect the libraries to be namespaced to e.g. `manifold::manifold`, `manifold::polygon` etc.

Reply:
Yes, I have packaged manifold for NixOS, and as soon as we can have a` find_package()` (#5284), it will work to compile-against and link the system-provided manifold.

Reply:
Cool! This PR implements the call to `find_package()`, so that part is sorted. I'm curious how you make the manifold library available in your package; e.g. do you provide a pkg-config .pc file, do you provide a CMake Config macro etc.?

Reply:
Yes :)
Manifold provides both of them, cmake and since recently also pkg-config (https://github.com/elalish/manifold/pull/901 ).

I guess `find-package()` will pick up the cmake macros.

Reply:
Btw., I believe the manifold pkg-config only supports linking with the core manifold library. It won't pick up the polygon and cross-section libraries exported by manifold.

Reply:
Yeah, that might need some refinement.

Reply:
Also opened https://github.com/elalish/manifold/issues/936 to discuss namespacing.

Reply:
Resolving this conversation for now as it's out of scope for this PR.